package.json

```
@@ -155,7 +155,7 @@
     "@types/webpack-hot-middleware": "^2.25.6",
     "@types/webpack-merge": "^5.0.0",
     "@types/xml2js": "^0.4.0",
-    "electron": "19.0.0",
+    "electron": "20.0.0",
```

Oh and this should be 22.0.0, so maybe this is more WIP than I thought it was? 😆 
```suggestion
    "electron": "22.0.0",
```

---

Woops! - yeah I kind of put this one together is a hurry at end of my day after doing 21.3.3. (But also didn't know if we wanted this yet - was going to check in with team next time we have everyone)